charts/skypilot/values.yaml

```
@@ -0,0 +1,116 @@
+apiService:
+  image: berkeleyskypilot/skypilot-nightly:latest
+  # Number of replicas to deploy - replicas > 1 is not well tested, and requires 
+  # a PVC that supports ReadWriteMany (see accessMode in storage section below).
+  replicas: 1
+  preDeployHook: |-
+    # Run commands before deploying the API server, e.g. installing an admin
+    # policy. Remember to set the admin policy in the config section below.
+    echo "Pre-deploy hook"
+    
+    # Uncomment the following lines to install the admin policy
+    
+    # echo "Installing admin policy"
+    # pip install git+https://github.com/michaelvll/admin-policy-examples
+
+  
+  # Set config.yaml content on the API server
+  config: null
+  # config: |
+  #   admin_policy: admin_policy_examples.AddLabelsPolicy
+  #
+  #   kubernetes:
+  #     allowed_contexts:
+  #       - gke_sky-dev-465_us-central1-c_skypilot-test-cluster
+  #       - gke_sky-dev-465_us-central1-c_skypilot-test-cluster-2
+  #
+  #   jobs:
+  #     controller:
+  #       resources:
+  #         cpus: 2+
+  #
+  #   allowed_clouds:
+  #     - aws
+  #     - kubernetes
+
+  # Set resource requests and limits for the API server
+  # TODO: Setting limits causes issues with the API server. For now we advise
+  # not to set limits.
+  resources: null
+  # resources:
+  #   requests:
+  #     cpu: "1"
+  #     memory: "2Gi"
+  #   limits:
+  #     cpu: "2"
+  #     memory: "4Gi"
+  #     # Additional resource types like ephemeral-storage can be specified here
+  
+  # [Internal] Enable developer mode for SkyPilot
+  skypilotDev: false
+
+storage:
+  # Enable/disable persistent storage
+  enabled: true
+  # Storage class name - leave empty to use cluster default
+  storageClassName: ""
+  # Access modes - ReadWriteOnce or ReadWriteMany depending on what is supported by the storage class
+  accessMode: ReadWriteOnce
+  # Storage size
+  size: 10Gi
+  # Optional selector for matching specific PVs
+  selector: {}
+    # matchLabels:
+    #   environment: prod
+  # Optional volume name for binding to specific PV
+  volumeName: ""
+  # Optional annotations
+  annotations: {}
+
+awsCredentials:
+  enabled: false
+
+gcpCredentials:
+  enabled: false
+  # TODO(romilb): This can be made optional by using the project in the key json by default.
+  projectId: null
+
+ingress:
+  enabled: true
+  # Name of the secret containing basic auth credentials for ingress. If not specified, a new secret will be created using authCredentials
+  authSecret: null
+  # Basic auth credentials in format "username:encrypted_password" (only used if ingressAuthSecret is not set)
```

minor: Not sure what is the convention for the naming, but the following might be a bit clearer?
```suggestion
  # Basic auth credentials in format "username:encrypted_password" (only used if ingress.authSecret is not set)
```